FileManager.php

```diff
@@ -5,8 +5,10 @@
 namespace MakinaCorpus\FilechunkBundle;
 
 use MakinaCorpus\FilechunkBundle\StreamWrapper\LocalStreamWrapper;
+use Symfony\Component\Finder\Finder;
```

Comment:
I'm not against this dependency, but please add the composer dependency as well. Try to keep it loose `^3.2 || ^4`.

Reply:
done